requirements.txt

```diff
@@ -0,0 +1,10 @@
+#django-netjsonconfig>=0.11.0,<0.13.0
+# temporary until django-netjsonconfig 0.12.0 is released
+git+https://github.com/openwisp/django-netjsonconfig@master#egg=django_netjsonconfig
```

Comment:
I see that you're putting this here, but then skipping it in `get_install_requires`, then this is also repeated in .travs.yml, which is not good, because we'll have to remember to update this twice, which some contributor may forget.

Maybe the `install-dev.sh` script makes sense. We can put this information in one place and explain in the README to call that script when doing development. I think we can keep that script. I think @atb00ker proposed it.

Reply:
:+1: for the idea, I remember some new contributors having problem because of this during GCI, mentioning a `install-dev.sh` should be helpful for them. :smile:

Reply:
Done!